cmds/group/actors.go

```diff
@@ -1,14 +1,14 @@
 package group
 
 import (
-	"context"
 	"fmt"
 	"io"
 	"sync"
 
 	"github.com/spf13/pflag"
 	tcclient "github.com/taskcluster/taskcluster-client-go"
 	"github.com/taskcluster/taskcluster-client-go/queue"
+	"golang.org/x/net/context"
```

Comment:
Yes, we used to simply import `"context"` instead of that whole address. I changed it to `"golang.org/x/net/context"`, as it's the same package, but compatible with older versions of Go.

I've scheduled for us to discuss which version of Go should be required to compile the project at our next meeting.